contracts/modules/PoolController.sol

```diff
@@ -142,30 +282,41 @@ contract PoolController is
             );
     }
     
-    function riskpools() external view returns(uint256 idx) { return _riskpools.length; }
+    function riskpools() external view returns(uint256 idx) { return _riskpoolIds.length; }
+
+
+    function getRiskpool(uint256 riskpoolId) public view returns(IPool.Pool memory riskPool) {
+        riskPool = _riskpools[riskpoolId];
+        require(riskPool.createdAt > 0, "ERROR:POL-020:RISKPOOL_NOT_REGISTERED");
+    }
+
 
     function getRiskpoolId(uint256 idx) 
         external view 
         returns(uint256) 
     { 
-        require(idx < _riskpools.length, "ERROR:POL-020:INDEX_TOO_LARGE");
-        return _riskpools[idx]; 
+        require(idx < _riskpoolIds.length, "ERROR:POL-021:INDEX_TOO_LARGE");
+        return _riskpoolIds[idx]; 
     }
 
     function getRiskPoolForProduct(uint256 productId) external view returns (uint256 riskpoolId) {
         return _riskpoolIdForProductId[productId];
     }
 
-    function _getRiskpool(IPolicy.Metadata memory metadata) internal view returns (IRiskpool riskpool) {
+    function getFullCollateralizationLevel() external view returns (uint256) {
+        return FULL_COLLATERALIZATION_LEVEL;
+    }
+
+    function _getRiskpoolComponent(IPolicy.Metadata memory metadata) internal view returns (IRiskpool riskpool) {
         uint256 riskpoolId = _riskpoolIdForProductId[metadata.productId];
-        require(riskpoolId > 0, "ERROR:POL-021:RISKPOOL_DOES_NOT_EXIST");
+        require(riskpoolId > 0, "ERROR:POL-022:RISKPOOL_DOES_NOT_EXIST");
 
         riskpool = _getRiskpoolForId(riskpoolId);
     }
 
     function _getRiskpoolForId(uint256 riskpoolId) internal view returns (IRiskpool riskpool) {
```

Comment:
`getRiskpool` and `_getRiskpoolForId` sound quite similar, but one returns a `IRiskpool` and the other a `IPool.Pool`. This is confusing.